go.mod

```diff
@@ -15,19 +15,19 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/blang/semver/v4 v4.0.0 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
-	github.com/go-logr/logr v1.2.3 // indirect
+	github.com/go-logr/logr v1.2.4 // indirect
 	github.com/golang/protobuf v1.5.3 // indirect
-	github.com/matttproud/golang_protobuf_extensions v1.0.2 // indirect
-	github.com/prometheus/client_golang v1.14.0 // indirect
-	github.com/prometheus/client_model v0.3.0 // indirect
-	github.com/prometheus/common v0.37.0 // indirect
-	github.com/prometheus/procfs v0.8.0 // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.4 // indirect
+	github.com/prometheus/client_golang v1.16.0 // indirect
+	github.com/prometheus/client_model v0.4.0 // indirect
+	github.com/prometheus/common v0.44.0 // indirect
+	github.com/prometheus/procfs v0.10.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/net v0.9.0 // indirect
-	golang.org/x/sys v0.7.0 // indirect
-	golang.org/x/text v0.9.0 // indirect
+	golang.org/x/net v0.12.0 // indirect
+	golang.org/x/sys v0.10.0 // indirect
+	golang.org/x/text v0.11.0 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20230525234030-28d5490b6b19 // indirect
 	google.golang.org/protobuf v1.30.0 // indirect
-	k8s.io/apimachinery v0.27.0 // indirect
+	k8s.io/apimachinery v0.28.0-rc.0 // indirect
 	k8s.io/component-base v0.27.0 // indirect
```

Comment:
Hmm weirdly component-base is still 0.27.0, it should be https://github.com/kubernetes/component-base/tree/v0.28.0-rc.0

Reply:
This is updated when I run `go mod tidy`, I tried updating this manually to `v0.28.0-rc.0` but the presubmits failed stating 
```
ERROR: go module files *not* up-to-date, they did get modified by 'GO111MODULE=on go mod tidy':
diff --git a/go.sum b/go.sum
```

Reply:
@xing-yang Does component-base sometimes go 1 version behind during rc update?

Reply:
Can you check which component is using component-base lib?  This is a indirect reference.

Reply:
The csi-lib-utils is using component-base but it is using 0.26.0

Reply:
Actually it has this replace and it is using 0.27.0, let me update that quickly

Reply:
After the tag is created for v0.15.0-rc.0, please change the above csi-lib-utils to v0.15.0-rc.0, and then this component-base should be able to use v0.28.0-rc.0

Reply:
Talked with @msau42 please use this tag: https://github.com/kubernetes-csi/csi-lib-utils/releases/tag/v0.15.0-rc0

We will change it to v0.15.0 soon for 1.28.0.

Reply:
Updated the package version